.github/pull_request_assignment.yml

```diff
@@ -1,165 +1,207 @@
 ---
+        
 - rule:
-    # translator data-plane PR
     paths:
-      - "specification/cognitiveservices/data-plane/TranslatorText/**"
+      - "specification/@(applicationinsights|operationalinsights)/**"
```

Comment:
The path should scope to /data-plan/, otherwise it contains data plan and management plane.